src/main/java/com/targomo/client/api/request/config/RequestConfigurator.java

```diff
@@ -199,7 +199,8 @@ private static void addMultiGraphLayer(TravelOptions travelOptions, JSONObject m
         if( Stream.of(travelOptions.getMultiGraphLayerType(), travelOptions.getMultiGraphLayerEdgeAggregationType(),
                 travelOptions.getMultiGraphLayerGeometryDetailPerTile(), travelOptions.getMultiGraphLayerMinGeometryDetailLevel(),
                 travelOptions.getMultiGraphLayerMaxGeometryDetailLevel(),
-                travelOptions.getMultiGraphLayerGeometryDetailLevel()).anyMatch(Objects::nonNull) ) {
+                travelOptions.getMultiGraphLayerGeometryDetailLevel(),
+                travelOptions.getMultigraphLayerCustomGeometryMergeAggregation()).anyMatch(Objects::nonNull) ) {
```

Comment:
you also need to add that in the if statement on line 68

Reply:
I have added locally, but as I saw that the other layer attributes weren't there, I removed before committing. Going to add again.